### PR TITLE
Reports a day counter

### LINKED ISSUE #1616: "Reports a day" counter is reset if the application is restarted
It may restart due to a) health b) a new release c) changing the environment variables.

In that case currently we lose our counter

A better way is to, at app startup, load the variable by doing a count of submissions already received that day thus far.

### PR DESCRIPTION
**Pull request by kevinyang06**
Fixes #1616 

# Description

> Please include a summary of the change.

# Any new packages installed?

> "Reports a day" counter is reset if the application is restarted

# Required followup work

> followup work required for query and async

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code

### CODE REVIEW

**Comment by kevinyang06** [comment]
It is weird that we do need `initializeAvailableData() `in order to make landing page redirect to CAFC

**Comment by kevinyang06** [comment]
I fixed a merge conflict for `server.js`, please make sure I didn't accidentally remove any useful code

File: f2/server.js [hunk 1]
```diff
@@ -78,15 +83,18 @@ const uploadData = async (req, res, fields, files) => {
...
   if (availableData.numberOfSubmissions >= process.env.SUBMISSIONS_PER_DAY) {
+    console.log('number of submissions: ' + availableData.numberOfSubmissions)
     console.log('Warning: redirecting request to CAFC')
     res.redirect(
       req.subdomains.includes('signalez')
         ? 'http://www.antifraudcentre-centreantifraude.ca/report-signalez-fra.htm'
         : 'http://www.antifraudcentre-centreantifraude.ca/report-signalez-eng.htm',
     )
   } else {
+    console.log('number of submissions: ' + availableData.numberOfSubmissions)
```
Comment (sastels) [inline f2/server.js]: You can 🔥 these two new console logs

File: f2/src/utils/saveRecord.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
 const MongoClient = require('mongodb').MongoClient
-
+const date = new Date()
+const currentDate =
+  (date.getDate() > 9 ? date.getDate() : '0' + date.getDate()) +
+  '/' +
+  (date.getMonth() > 8 ? date.getMonth() + 1 : '0' + (date.getMonth() + 1)) +
+  '/' +
+  date.getFullYear()
+console.log(currentDate)
```
Comment (sastels) [inline f2/src/utils/saveRecord.js]: 🔥 this log

File: f2/src/utils/saveRecord.js [hunk 3]
```diff
@@ -44,5 +52,33 @@ async function saveRecord(data, res) {
...
+              $eq: currentDate,
+            },
+          })
+          .toArray(function(err, result) {
+            if (err) {
+              console.warn(`ERROR in find: ${err}`)
+            } else {
+              db.close()
+              numberofReports = result.length
+              console.log('number of reports submitted: ' + numberofReports)
```
Comment (sastels) [inline f2/src/utils/saveRecord.js]: 🔥 this log

File: f2/server.js (hunk 1 above)
Comment (kevinyang06) [reply f2/server.js]: done

File: f2/src/utils/saveRecord.js (hunk 3 above)
Comment (kevinyang06) [reply f2/src/utils/saveRecord.js]: done

File: f2/src/utils/saveRecord.js (hunk 2 above)
Comment (kevinyang06) [reply f2/src/utils/saveRecord.js]: done

**Pull request merged**